.github/workflows/ci-core-reusable.yml

```diff
@@ -99,13 +99,16 @@ jobs:
           ci_run git config --global --add safe.directory /usr/src/zksync/contracts
 
           ci_run zk
-          ci_run zk init
+          ci_run zk init --chain-id-hack
```

Comment:
nit: I don't like that we're calling it 'chain-id-hack'. We should have more descriptive name

Reply:
@mm-zk I am not sure I can come up with better name without it being too bloated. I could have something like `chain-id-same-as-era-chain-but-incorrect-era-chain-id-on-l2-bridge`. The description itself can be currently read in the `zk` tool:

```
        '--chain-id-hack',
        'used to test LegacyBridge compatibily. The chain will have the same id as the era chain id, while eraChainId in L2SharedBridge will be 0'
```

Considering that this is a temporary feature used only for local development until the full migration the shared bridge is complete, I would stick to this name and delete it afterwards

Reply:
ah, I see now, sorry